src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -153,16 +165,11 @@ class BaseFilterInput<T> extends Component<FilterInputProps<T>> {
 }
 
 const FilterInputWithoutTokens: <T>(
```

Comment:
This should be renamed as there are no more tokens, probably leftovers from earlier changes.

Reply:
Might have been better to indicate a type cast in the name as it is now the only function for the renamed FilterInputWithoutTokens (UnstyledFilterInput). But it's ok for me as it is now.

Reply:
Looks good now!